kernel/src/engine/parquet_stats_skipping.rs

```diff
@@ -1,9 +1,8 @@
 //! An implementation of data skipping that leverages parquet stats from the file footer.
-use crate::expressions::{BinaryOperator, Expression, Scalar, UnaryOperator, VariadicOperator};
+use crate::expressions::{
+    BinaryOperator, ColumnName, Expression, Scalar, UnaryOperator, VariadicOperator,
+};
 use crate::schema::DataType;
-// TODO: This struct is this module's only dependency on arrow-parquet. Once kernel has proper
-// support for nested paths, convert this code to use that instead.
-use parquet::schema::types::ColumnPath;
```

Comment:
I don't _think_ this change has any immediate consequences, but it _does_ potentially allow us to make the code available for non-parquet use cases (e.g. https://github.com/delta-incubator/delta-kernel-rs/pull/420)

Reply:
#379